src/components/organisms/about/about.scss

```
@@ -0,0 +1,33 @@
+@use "../../variables" as *;
+
+.container {
+  padding: 0px;
+  margin: 100px 0px;
+  @media screen and (max-width: $breakpoint-lg) {
+    margin: 40px 0px;
+  }
+}
+.imgContent {
+  display: flex;
+  @media screen and (max-width: $breakpoint-lg) {
+    flex-wrap: wrap;
+  }
+  img {
+    width: 25%;
+    height: auto;
+    @media screen and (max-width: $breakpoint-lg) {
+      width: 50%;
+    }
+  }
+}
+.top-message {
+  width: 100%;
+  background-color: $white_smoke;
+}
+
+.title {
+  color: $orange_bistre;
+}
+.explain {
+  color: $black;
+}
```

```suggestion
.top-message {
  width: 100%;
  background-color: $white_smoke;
  .title {
  color: $orange_bistre;
  }
  .explain {
  color: $black;
  }
}
```
厳密にsassの書き方に習うならば、上記のような入れ子構造になります。さらに厳密なことを言うと、`.container`の中に`.imgContent`以下のすべての記述が入る構造になります。
ただし、最近の潮流として入れ子にせずベタでcssを書くことも増えているため、この点に関しては修正してもしなくてもOKです。